.gitignore

```diff
@@ -1,27 +1,9 @@
-.classpath
```

Comment:
Why did you change the gitignore? That's unrelated to the changes that should be in this pull request.